### PR TITLE
format and clean up code, tweak gradient and remove page buttons on

### LINKED ISSUE #59: fix responsive design for footer and pageButtons
image

### PR DESCRIPTION
**Pull request by adamkumlin**
small screens.

clicking on the small page buttons is challenging enough on a small screen. in my opinion it is better to just remove them entirely rather than making them even smaller. i am however open to suggestions and other opinions

closes #59

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -257,16 +262,15 @@ main div #enlarged-image {
...
 main>#enlarged-image-div>#enlarged-image-go-back-button {}
 
+/* What do we do with this? */
+
 #navigationButtons {
     width: fit-content;
     margin-right: auto;
     margin-left: auto;
     margin-bottom: 2rem;
 }
```
Comment (simon-s-99) [inline style.css:272]: ^ this is old code from when we were using the arrow icons (.svg)
feel free to remove :)

File: index.html [hunk 2]
```diff
@@ -24,111 +24,114 @@
...
+  <input type="submit" value="Sök" id="submit" />
+</form>
+
+<main>
+  <!-- [PLACE PICTURE RESULTS HERE] -->
+</main>
+
+<div id="navigationButtons">
+  <button type="button" id="previous">
+    << /button>
```
Comment (simon-s-99) [inline index.html:125]: remove this, it is causing this V

image

File: style.css [hunk 3]
```diff
@@ -365,6 +373,10 @@ footer {
         left: 3.7rem;
     }
 
+    #pageButtons {
+        display: none;
+    }
```
Comment (simon-s-99) [inline style.css:378]: Currently the numbered page buttons do not show up on the first page of desktop version, the problem is most likely  here or somewhere else in the css as i can see no major changes to the script.js file

image

**Review by simon-s-99 — CHANGES_REQUESTED** [review]
Overall i like the decision to remove the buttons on mobile, good call 👍 

Also, could we make the numbered page buttons just a little smaller on desktop? 🙂
image

File: index.html (hunk 2 above)
Comment (adamkumlin) [reply index.html:125]: the << /button> seems to be the culprit here, should be fixed now

File: style.css (hunk 3 above)
Comment (adamkumlin) [reply style.css:378]: same, see above

**Comment by adamkumlin** [comment]
> Overall i like the decision to remove the buttons on mobile, good call 👍
> 
> Also, could we make the numbered page buttons just a little smaller on desktop? 🙂 image

the << /button> seems to be the culprit here, should be fixed now

**Comment by simon-s-99** [comment]
good job 💯

**Comment by simon-s-99** [comment]
i miss-clicked and did not mean to close 🤣

**Pull request merged**